crates/aptos-rosetta/src/account.rs

```diff
@@ -179,7 +182,14 @@ async fn get_balances(
             })
             .collect())
     } else {
-        Ok(HashMap::new())
+        let mut currency_map = HashMap::new();
+        currency_map.insert(
+            native_coin_tag(),
```

Comment:
This makes me wonder if we should just do this in both arms. As in, do something like:
```
currency_map.entry(native_coin_tag()).or_insert_with(|| Balance {coin: TestCoin {value: U64(0) }});
```
Just before the return.

Might be weird to assume we have `native_coin_tag` by default only in some cases. Correct me if the previous block is doing something I'm missing.